include/antler/system/utils.hpp

```diff
@@ -86,6 +86,7 @@ namespace antler::system {
       for (const auto& arg : args) {
          cmd += " " + arg;
       }
+      cmd += " 2>&1";

```

Comment:
what does it affect? if there will be error you return `nullopt` and if not you'll unlikely to have error output at all. And even if you do, parsing of output will likely break.

Reply:
It resolves issue: `sh: 1: gh: not found` in the original issue.

I believe Bucky's intention was to run command line `gh --version` quietly (the function name is `execute_quiet`).
While stdout was silenced, the stderr was still printing error (which was ignored anyway).

Reply:
I doubt that could be a case. if we will have non-critical error output on zero return code that may corrupt parsing of a valid data. Ideally that should be similar to `subprocess.run` in python. i.e. stdout and stderr piping is customizable. but I would leave it to next issue if that will be really needed. If you believe your case is more likely to happen than mine, just leave comment like `pipe error output from console to stdout so caller can parse error string`

Reply:
Sure, done